<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="22271" started-at="2017-08-29T02:25:26Z" finished-at="2017-08-29T02:25:48Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="22271" started-at="2017-08-29T02:25:26Z" finished-at="2017-08-29T02:25:48Z">
      <class name="day24.com.automationtesting.extentreports.CapturingScreenshot">
        <test-method status="PASS" signature="config()[pri:0, instance:day24.com.automationtesting.extentreports.CapturingScreenshot@8bd1b6a]" name="config" is-config="true" duration-ms="280" started-at="2017-08-29T07:55:26Z" finished-at="2017-08-29T07:55:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- config -->
        <test-method status="FAIL" signature="captureScreenshot()[pri:0, instance:day24.com.automationtesting.extentreports.CapturingScreenshot@8bd1b6a]" name="captureScreenshot" duration-ms="20225" started-at="2017-08-29T07:55:26Z" finished-at="2017-08-29T07:55:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Home-Selenium Webdriver Appium Complete Tutorial] but found [Home - Automation Test]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Home-Selenium Webdriver Appium Complete Tutorial] but found [Home - Automation Test]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at day24.com.automationtesting.extentreports.CapturingScreenshot.captureScreenshot(CapturingScreenshot.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- captureScreenshot -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:day24.com.automationtesting.extentreports.CapturingScreenshot@8bd1b6a]" name="getResult" is-config="true" duration-ms="1183" started-at="2017-08-29T07:55:47Z" finished-at="2017-08-29T07:55:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=captureScreenshot status=FAILURE method=CapturingScreenshot.captureScreenshot()[pri:0, instance:day24.com.automationtesting.extentreports.CapturingScreenshot@8bd1b6a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:day24.com.automationtesting.extentreports.CapturingScreenshot@8bd1b6a]" name="endReport" is-config="true" duration-ms="556" started-at="2017-08-29T07:55:48Z" finished-at="2017-08-29T07:55:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- day24.com.automationtesting.extentreports.CapturingScreenshot -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
